index.html

```diff
@@ -0,0 +1,6349 @@
+<!Doctype html>
+<html>
+<title>Canvas Palette</title>
+<meta charset='utf-8'>
+<head>
+    <title>imgExample</title>
+    <script src="data/4x4.json"></script>
+    <script src="data/32x32.json"></script>
+    <link href="style.css" rel="stylesheet"/>
+</head>
+<body>
+<header>
+    <h1>CodeJam - Pallete</h1>
+</header>
+<main>
+    <div>
+        <div class="menu">
+        <section class="section-1">
+            <ul>
+                <li>Paint bucket</li>
+                <li>Choose color</li>
+                <li>Move</li>
+                <li>Transform</li>
+            </ul>
+        </section>
+        <section class="section-2">
+            <ul>
+                <li>Current color</li>
+                <li>Prev color</li>
+                <li>red</li>
+                <li>blue</li>
+            </ul>
+        </section>
+    </div>
+    <div>
+        <form>
+            <p>Please select size:</p>
+            <input type="radio" name="picture" value="32x32" onclick="handleClick(this);">32x32<br>
```

Comment:
dont use onclick, use addEventListener in js

Reply:
And use delegation https://learn.javascript.ru/event-delegation

Reply:
totally agree